app/move/controllers/view/view.update.links.js

```diff
@@ -0,0 +1,26 @@
+const getUpdateLinks = (updateSteps, t, urls) => {
```

Comment:
Rather than having to pass in `t` you can directly import the [i18n module](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/5f692a046faea2298687c7a23bc4882734c7a60b/config/i18n.js) and call `i81n.t` on that.

Here's [an example](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/5f692a046faea2298687c7a23bc4882734c7a60b/common/presenters/move-to-card-component.js#L1).

Reply:
Sure, but there is a slight werdness there. I had done that and got bitten by using 

```
const { t } = require('../../../../config/i18n')
```

and getting

```
error: Cannot read property 'translator' of undefined {"stack":"TypeError: Cannot read property 'translator' of undefined\n    at t
```

whereas _obviously_ I should have been doing ;)

```js
const i18n = require('../../../../config/i18n')

module.exports = () => {
...
   const fooStr = i18n.t('foo')
}
```

Maybe it's worth waiting for i18n to actually be properly instantiated before it gets exported. Not a biggy, but principle of least surprise...

Reply:
I also find the second approach is much testable as you can use sinon to stub the `t` method. Whereas if importing using destructuring you would have to use something lik proxyquire to stub and it gets messy...